src/display/api.js

```diff
@@ -632,74 +688,88 @@ class PDFDocumentProxy {
 
   /**
    * @param {string} id - The named destination to get.
-   * @returns {Promise} A promise that is resolved with all information
-   *   of the given named destination.
+   * @returns {Promise<Array<any>>} A promise that is resolved with all
+   *   information of the given named destination.
    */
   getDestination(id) {
     return this._transport.getDestination(id);
   }
 
   /**
-   * @returns {Promise} A promise that is resolved with an {Array} containing
-   *   the page labels that correspond to the page indexes, or `null` when
-   *   no page labels are present in the PDF file.
+   * @returns {Promise<Array<string> | null>} A promise that is
+   *   resolved with an {Array} containing the page labels that correspond to
+   *   the page indexes, or `null` when no page labels are present in the PDF
+   *   file.
```

Comment:
This first line is now weirdly short, compared to the rest of them, it looks like this was incorrectly re-formatted.

Reply:
I think so. I'll address that in the upcoming patch too.